aiida/restapi/translator/nodes/node.py

```diff
@@ -563,10 +563,7 @@ def get_formatted_result(self, label):
 
         for node_entry in results[result_name]:
             # construct full_type and add it to every node
-            try:
-                node_entry['full_type'] = construct_full_type(node_entry['node_type'], node_entry['process_type'])
-            except KeyError:
-                node_entry['full_type'] = None
+            node_entry['full_type'] = construct_full_type(node_entry.get('node_type'), node_entry.get('process_type'))
```

Comment:
It'll result (at the very minimum) in a `full_type` value of `"|"`.

Reply:
Also, if it didn't work the tests would fail...

Reply:
> Also, if it didn't work the tests would fail...

If you're sure that this always works (either all note types tested or for other reason) that's fine. Just wanted to be sure, since you're changing the behavior here.

Reply:
Yeah. I guess this change is not really relevant for this PR (within the scope of it), so I could technically remove this change for now.

Reply:
If you're unsure whether this can cause problems upstream (thinking e.g. of things like the provenance browser, etc.), perhaps it's safer not to change it here.